### ISSUE TITLE
Exporter release?

### ISSUE THREAD

**Issue by gtherond** [mrlhansen/idrac_exporter#81]
Hi! Would it be possible to get the exporter binary release? Something similar to what node_exporter do would be wonderful.

We're currently cloning and building it from source but we're on a really constrained environment and it would be much easier for us to be able to just download the binary release of the exporter instead of having to build it from scratch.

I think it would be possible to use GA to achieve that isn't it?

Thanks a lot!

**Comment by mrlhansen** [comment]
Hi @gtherond 

I'm sure it can be done with GA, I will look into it (might be a few weeks, I'm about to go on vacation).

**Comment by gtherond** [comment]
aaaaah cool! you rocks!
TBN: I'm currently integrating the exporter on Openstack kolla/kolla-ansible lifecycle project, it will be much quicker for our operators/users to indeed directly call for the final github binary release rather than the sources and then build the go project as some of them are in a constrained air-gapped environment where building binaries at runtime require much more data than just having the exporter source-code (Except if the exporter does provide the vendor folder of course).

Thanks a lot!

**Comment by mrlhansen** [comment]
I am happy to announce that the first release with pre-built binaries is now out :tada: 
* https://github.com/mrlhansen/idrac_exporter/releases/latest

**Comment by gtherond** [comment]
cool! thanks a lot! I'll test that!

Quick suggestions on the side:
As your exporter is able to request multiple management systems as long as they're redfish endpoint, a more straight forward name would be something like:

- bmc_exporter
- redfish_exporter
- redfish_bmc_exporter


That would immediately explain the purpose ^^ But all in all that's not that of a burden, thank you very much for your work!

**Comment by mrlhansen** [comment]
Yes, I also thought about changing the name, but I feel it might be a little late now.

**Comment by gtherond** [comment]
Yeah no worry, you can change that on 1.4.0 release I guess :D
